lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -102,6 +102,7 @@
                                     </mat-icon>
                                     <ng-template #no_selected_row>
                                         <img
+                                            [attr.aria-label]="iconType(data.getValue(row, col))"
```

Comment:
we should probably have a pipe instead of the method `iconType`. @DenysVuika

Reply:
yeah, in terms of performance, the pipe will be much faster